server/plugins/TriggerUGens.cpp

```diff
@@ -893,14 +892,56 @@ void SetResetFF_next_k(SetResetFF* unit, int inNumSamples) {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////
 
```

Comment:
It is a bit a matter of taste, but I would find it more readable if we just had three functions without a template, and choose the right one from Ctor?

Reply:
For me the main advantage is that it makes it more clear what code is in common and what differs, and it also keeps the different parts closer together. For UGens where the diverging code is small, I also find it less error prone.

On a second thought, it would be probably better to have this discussion in a separate place and focus on ToggleFF here. The template is also not so necessary for a UGen so small and without many alternative calc functions, so I reverted to the more traditional style.